web/modules/custom/portland/portland.module

```diff
@@ -839,4 +889,44 @@ function portland_form_alter(&$form, &$form_state, $form_id) {
             }
         }
     }
+    else if(substr_compare($form_id, $group_form_prefix, 0, strlen($group_form_prefix)) === 0) {
```

Comment:
I'm thinking this is a lot of code in the overall portland.module. Should we use this as an opportunity to move some of the group functions into a submodule? 🤔

Reply:
Probably will do that as another story. This one is getting complicated now. :)